libs/datasets/timeseries.py

```diff
@@ -133,8 +128,32 @@ def get_subset(
 
         return self.__class__(data)
 
+    def get_data_for_fips(self, fips) -> List[dict]:
+        """Get data for FIPS code.
+
+        Args:
+            fips: FIPS code.
+
+        Returns: List of dictionary records with NA values replaced to be None
+        """
+
+        pd_data = self.get_subset(None, fips=fips).data
+        pd_data = pd_data.where(pd.notnull(pd_data), None)
+        return pd_data.to_dict(orient="records")
+
+    def get_data_for_state(self, state) -> List[dict]:
```

Comment:
It's just that the term `data` tends obfiscate the underlying object type.

```suggestion
    def get_records_in_state(self, state) -> List[dict]:
```